style.css

```diff
@@ -328,4 +338,272 @@ button:hover {
 
 .app_store{
     margin-left: 24px;
+}
+
+/*.none {*/
+/*    display: none !important;*/
+/*}*/
+
+@media screen and (max-width: 768px) {
+
+    section{
+        display: none;
+    }
+
+    .advantages-header {
+        text-align: center;
+        margin-top: 24px;
+        margin-bottom: 32px;
+        font-family: "Roboto", sans-serif;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 28px;
+        line-height: 3;
+        color: #383838;
+    }
+
+
+    hr {
+        display: none;
+    }
+
+    .advantages-cards-wrapper {
+        margin: 0 104px 0 104px;
+        display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .card {
+        max-width: 296px;
+        background-color: #ffff;
+        display: flex;
+        /*flex-direction: column;*/
+        justify-content: center;
+        align-items: center;
+        margin-bottom: 32px;
+    }
+
+    .card:last-child{
+        margin-bottom: 40px;
+    }
+
+    .card-img {
+        margin-bottom: 16px;
+        transform: scale(70%);
```

Comment:
If you need to use smaller image, just use appropriate width and height instead of transform